plugins/importexport/medra/templates/crDepositErrors.tpl

```diff
@@ -0,0 +1,62 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
+	 "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
+<html xmlns="http://www.w3.org/1999/xhtml">
+	<head>
+		<meta http-equiv="Content-Type" content="text/html; charset={$defaultCharset|escape}" />
+		<style type="text/css">
+			table thead th {
+				border: 1px solid black; 
+				font-size: 18; 
+				font-weight: bold; 
+				padding: 10px;
+			}
+			table {
+				border-collapse: collapse;
+				font-size: 18;
+			}
+			table tbody td {
+				border: 1px solid black; 
+				font-size: 18; 
+				padding: 10px;
+			}
+			
+			table tbody tr:nth-child(even) {
+				background: #ABCDEF;
+			}
+			table tbody tr:nth-child(odd) {
+				background: #BDB9AA;
+			}
+			
+		</style>
+	</head>
+	<body>
+		<h2> {translate key="plugins.importexport.common.validationErrors"} </h2>
+		
+		<div> {translate key="plugins.importexport.medra.crossref.error.cause"} </div> <br/>
+		<div> {translate key="plugins.importexport.medra.crossref.error.number"}:  {$numberError} </div> <br/>
+		<div> {translate key="plugins.importexport.medra.crossref.error.details"}: </div> <br/>
+		<table>
+			<thead>
+				<tr>
+					<th>{translate key="plugins.importexport.medra.crossref.error.code"}</th>
+					<th>{translate key="plugins.importexport.medra.crossref.error.element"}</th>
+					<th>{translate key="plugins.importexport.medra.crossref.error.description"}</th>
+				</tr>
+				
+			</thead>
+			<tbody>
+				{foreach from=$headlines key=k item=v}
+					<tr>
+						<td>{$v['code']}</td>
```

Comment:
Will these next three lines be displaying "safe" content? If they will be plain-text, then passing them through the Smarty `escape` modifier will ensure that special characters (like `&`, `<`, `>`) will be displayed properly (and any potential XSS risks will be mitigated). If they are going to contain legitimate HTML contents, then I'd recommend passing them through `strip_unsafe_html` to ensure that XSS risks are mitigated. (The list of allowed HTML elements is configured in `config.inc.php` so if using that I'd recommend checking that the defaults match against what's expected here.)

Reply:
I used escape o ensure that special characters will be dispayed properly

Reply:
> 
> 
> > @njikonga-med, my apologies for not following up on this sooner. I've added some additional comments after another review.
> > Are you still working with OJS 3.1.2? There are some changes in OJS 3.2.x that will require an update -- the addition of metadata versioning in particular.
> 
> I was on holiday.
> Do you mean, i have to switch from OJS 3.1.2 to OJS 3.2.x and apply my changes?

Hello my dear, how are you? Do you have news? it's URGENT to perform update of mEDRA plugin. Could you please let me know?
Thanks!!!